testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MsalSampleApp.java

```diff
@@ -40,27 +46,39 @@
  */
 
 public class MsalSampleApp extends Application {
-    private static final int LOG_SIZE = 1024 * 1024;
     private StringBuilder mLogs;
-    private int mLogSize;
+
+    private synchronized void writeToLogFile(Context ctx, String msg) {
+        File directory = ctx.getDir(ctx.getPackageName(), Context.MODE_PRIVATE);
+        File logFile = new File(directory, "msallogs.txt");
+        FileOutputStream outputStream = null;
+        try {
+            outputStream = new FileOutputStream(logFile, true);
```

Comment:
Are going to rollover this file when it becomes too large?  I'd be concerned about someone taking this code as is and copying it in to their application.